website/src/components/canvas/Sidebar.jsx

```diff
@@ -89,7 +112,18 @@ class Sidebar extends React.Component {
   mouseEnter() {
     this.setState({opacity: 1})
   }
-
+  onClickClose() {
+   this.setState(prevState => ({
+     isOpen: !prevState.isOpen,
+     closed: 1
+   }));
+ }
+ onClickOpen() {
```

Comment:
Thanks! You can also follow conventions in the other components that handle open & close, which I believe is something like handleSidebarOpen